certified-connectors/Eden AI/apiDefinition.swagger.json

```diff
@@ -0,0 +1,10511 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "Launch",
+    "description": "Enhance your apps and workflows with the power of Eden AI ! Integrate advanced AI features such as natural language processing, image recognition, document parsing, audio processing, generative AI and more. ",
```

Comment:
@queSaDiLLaSS all files should be properly formatted. please remove the white space at the end of the description.